src/workflows-app/utils/anonymous-username-pairs.js

```diff
@@ -0,0 +1,186 @@
+export const leftNames = [
+  'admiring',
+  'adoring',
+  'affectionate',
+  'agitated',
+  'amazing',
+  'angry',
+  'awesome',
+  'beautiful',
+  'blissful',
+  'bold',
+  'boring',
+  'brave',
+  'busy',
+  'charming',
+  'clever',
+  // 'compassionate',
+  'competent',
+  // "condescending",
+  'confident',
+  'cool',
+  'cranky',
+  'crazy',
+  'dazzling',
+  'determined',
+  'distracted',
+  'dreamy',
+  'eager',
+  'ecstatic',
+  'elastic',
+  'elated',
+  'elegant',
+  'eloquent',
+  'epic',
+  'exciting',
+  'fervent',
+  'festive',
+  'flamboyant',
+  'focused',
+  'friendly',
+  'frosty',
+  'funny',
+  'gallant',
+  'gifted',
+  'goofy',
+  'gracious',
+  'great',
+  'happy',
+  'hardcore',
+  // "heuristic",
+  'hopeful',
+  'hungry',
+  'infallible',
+  'inspiring',
+  'intelligent',
+  'interesting',
+  'jolly',
+  'jovial',
+  'keen',
+  'kind',
+  'laughing',
+  'loving',
+  'lucid',
+  'magical',
+  'modest',
+  'musing',
+  'mystifying',
+  // "naughty",
+  'nervous',
+  'nice',
+  'nifty',
+  'nostalgic',
+  'objective',
+  'optimistic',
+  'peaceful',
+  'pedantic',
+  'pensive',
+  'practical',
+  'priceless',
+  'quirky',
+  'quizzical',
+  'recursing',
+  'relaxed',
+  'reverent',
+  'romantic',
+  'sad',
+  'serene',
+  'sharp',
+  'silly',
+  'sleepy',
+  'stoic',
+  'strange',
+  'stupefied',
+  'suspicious',
+  'sweet',
+  'tender',
+  'thirsty',
+  'trusting',
+  'unruffled',
+  'upbeat',
+  'vibrant',
+  'vigilant',
+  'vigorous',
+  'wizardly',
+  'wonderful',
+  // "xenodochial",
+  'youthful',
+  'zealous',
+  'zen',
+];
+
+export const rightNames = [
```

Comment:
the original list used the last names of famous scientists and programmers. during feedback, someone asked if this list was inclusive of diverse populations, didn't include any known terrible people, etc. 

it's just easier to use animals and avoid all those issues (it's also a better aesthetic, in my opinion).

Reply:
I like the idea of using animal names. Do both these lists need to be run by UX?